frappe/core/doctype/user/user.py

```diff
@@ -785,30 +778,43 @@ def sign_up(email, full_name, redirect_to):
 
 		if redirect_to:
 			frappe.cache().hset('redirect_after_login', user.name, redirect_to)
+	else:
+		user = frappe.get_doc("User", email)
+		if not user.enabled:
+			site_url = get_url()
+			subject = _("ERPNext User Account Disabled")
```

Comment:
This might not be the case always...

Reply:
why not?

Reply:
The framework can be used with or without "ERPNext"...

Reply:
oh, I'll just remove ERPNext then...